sljit_src/sljitConfigInternal.h

```diff
@@ -126,7 +126,20 @@ extern "C" {
 /* Automatic CPU detection (requires compiler support). */
 /********************************************************/
 
-#if (defined SLJIT_CONFIG_AUTO && SLJIT_CONFIG_AUTO)
+#if !((defined SLJIT_CONFIG_X86_32 && SLJIT_CONFIG_X86_32) \
```

Comment:
The long `#if` above could handle this by defining SLJIT_CONFIG_AUTO if it is not defined and no other cpu is explicitly selected.

Reply:
it would throw some warnings if some value other than 0 or 1 is passed with SLJIT_CONFIG_AUTO, but looks nicer